### DISCUSSION TITLE
modifying in the code

### DISCUSSION THREAD

**Discussion by martten20** [stella-cv/stella_vslam#191] [Q&A] [answered]
hi,
I wanted to know how to apply the openvslam to live stream like in robots?

**Comment by ymd-stella** [comment, marked as answer]
Use https://github.com/OpenVSLAM-Community/openvslam_ros.
docs: https://openvslam-community.readthedocs.io/en/latest/ros.html

**Reply by martten20** [reply]
thanks alot.
